airbyte-integrations/connectors/source-google-ads/source_google_ads/google_ads.py

```diff
@@ -62,7 +63,7 @@ def send_request(self, query: str) -> SearchGoogleAdsResponse:
     @staticmethod
     def get_fields_from_schema(schema: Mapping[str, Any]) -> List[str]:
         properties = schema.get("properties")
-        return [*properties]
+        return list(properties.keys())
```

Comment:
the error will be raised, but this error will be raised as well in previous implementation ( `return [*properties]`) - we always had them in _predefined_ schemas in schemas folder.

Reply:
also, in case we have no properties in schema (like we used), I parse the raw query.
Getting attributes is mandatory, we cannot perform read properly without it.

Reply:
outdated, now everything read is taken from get_json_schema